system_tests/speech.py

```diff
@@ -125,6 +125,32 @@ def test_sync_recognize_gcs_file(self):
                                          max_alternatives=1)
         self._check_best_results(result)
 
+    def test_sync_recognize_local_file_rest(self):
+        with open(AUDIO_FILE, 'rb') as file_obj:
+            results = self._make_sync_request(content=file_obj.read(),
+                                              max_alternatives=2,
+                                              use_gax=False)
+            second_alternative = results[1]
+            self.assertEqual(len(results), 2)
```

Comment:
This check should come **before** you access `results[1]`, otherwise you'd get an `IndexError` instead of a failed assertion.